glide.lock

```diff
@@ -8,10 +8,11 @@ imports:
   subpackages:
   - spew
 - name: github.com/emicklei/go-restful
-  version: 858e58f98abd32bc4d164a08e8d7ac169ae876e2
+  version: dc0f94ee75de39d6420e5446b0222490264bb90f
   subpackages:
   - log
-  - swagger
+- name: github.com/emicklei/go-restful-openapi
+  version: efa082cf7dfa43fc8381ca3bfb9ef9a248c5d686
```

Comment:
Could you update to `7cb0644488653a4657f941eb34631edf7d5d21e3`? Otherwise we get a lot of validation errors.

Reply:
Done